cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -686,6 +686,7 @@ instance ( ConvertRawHash blk
                [ "kind" .= String "TraceAddBlockEvent.SwitchedToAFork"
                , "newtip" .= renderPointForVerbosity verb (AF.headPoint new)
                , "chainLengthDelta" .= new `chainLengthΔ` old
+               , "realFork" .= not (AF.withinFragmentBounds (AF.headPoint old) new)
```

Comment:
Maybe rename this to `"sanityCheck"`? Or add a comment explaining that we're just double-checking that the `SwitchedToAFork` event really was incurred by a proper fork.